app/user-status/client/admin/userStatusEdit.css

```diff
@@ -0,0 +1,3 @@
+.rc-select {
+    margin: .5rem 0;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4776501)